doc/lsst.ap.pipe/apdb.rst

```diff
@@ -16,20 +16,19 @@ Setting up the Alert Production Database for ap_pipe
 In its default configuration, the Alert Production Pipeline, as represented by :file:`pipelines/_ingredients/ApPipe.yaml`, relies on a database to save and load DIASources and DIAObjects.
 When running as part of the operational system, this database will be provided externally.
 However, during testing and development, developers can run |apdb-cli| to set up their own database.
-This page provides an overview of how to use |apdb-cli|.
-Note that this document applies to APDB implementation backed by SQL database (SQLite or PostgreSQL), Cassandra-based implementation will use different set of options.
+This page provides an overview of how to integrate the AP pipeline with |apdb-cli|.
 
 .. _section-ap-pipe-apdb-config:
 
 Configuring the database
 ========================
 
-The database is configured using `~lsst.dax.apdb.ApdbConfig`.
+The database is configured using |apdb-cli|, as described in the :ref:`dax_apdb documentation <lsst.dax.apdb-scripts>`.
+If you are creating your own database, |apdb-cli| outputs a config file encoding the database settings.
+If you are using a database set up by someone else, they should have provided you with a config file or its label in the ``dax_apdb`` index.
```

Comment:
Could you add another sentence explaining what you mean by `label in the ``dax_apdb`` index`?